Added 15+ new crates, fixed issues raised in #9901

Owai-Seek opened the pull request:
Remake of previous PR that I broke due to inexperience.  #9901 Also apparently #9905 needs this to work.
Added 15+ new crates to cargo.
Re-ordered some some of the crates.

Added the following:

/datum/supply_pack/misc/randombedsheets
/datum/supply_pack/misc/coloredsheets
/datum/supply_pack/misc/cbtpack
/datum/supply_pack/misc/exoticfootwear
/datum/supply_pack/misc/shower
/datum/supply_pack/misc/casinocrate
/datum/supply_pack/misc/coincrate
/datum/supply_pack/organic/fiestatortilla
/datum/supply_pack/organic/fakemeat
/datum/supply_pack/organic/mixedboxes
/datum/supply_pack/organic/monkeydripmeat
/datum/supply_pack/emergency/medemergencylite
/datum/supply_pack/emergency/medemergency
/datum/supply_pack/service/buildabar
/datum/supply_pack/service/maintgarden
/datum/supply_pack/service/janitor/advanced
/datum/supply_pack/critter/mouse
/datum/supply_pack/medical/bodybags

and probably a few more I may have forgotten about.

## About The Pull Request

In my attempt to clean up the categories in cargo I instead make the problem worse by adding even more crates to buy. (I'll probably be adding even more and moving stuff around at a later date, but I had to get these out before I burnt out on it.)

## Why It's Good For The Game

More Crates

## Changelog
:cl:
add: 15+ new crates for cargo
tweak: organizes crates and moving them to proper categories
spellcheck: fixed a few typos
del: some dumb stuff like toner crates
re
/:cl:

Owai-Seek committed:
Added 12 new crates, fixed issues raised in #9901

Remake of previous PR that I broke due to inexperiance.
Added 12 new crates to cargo.
Re-ordered some some of the crates.

Trilbyspaceclone commented on code/modules/cargo/packs/emergency.dm:
```diff
@@ -44,8 +44,28 @@
+/datum/supply_pack/emergency/medemergency
+	name = "Emergency Medical Supplies"
+	desc = "Emergency supplies for a front-line medic. Contains two boxes of body bags, a medical HUD, a defib unit, medical belt, toxin bottles, epipens, and several types of medical kits."
+	cost = 3000
```
oooh this seems like a good "DropPod Only"

Owai-Seek replied:
I went ahead with your suggestion but during testing I found out that drop pods cant be used with defib units. (at least the loaded variant). I was sick of testing at that point, so I boosted the price to 6600 and made it available for regular purchase.

Owai-Seek committed:
Made Medical Emergency Crate Drop-Pod only.

Trill made me do it.

Owai-Seek committed:
Added more crates.

Trilbyspaceclone commented on code/modules/cargo/packs/misc.dm:
```diff
@@ -143,6 +115,136 @@
+	crate_name = "coin crate"
+	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/misc/coincrate/generate()
```
Make them random and lower the cost to 4k~

Owai-Seek replied:
There are coins that can go up to 3000 credits. The crate isn't supposed to be particularly practical or worthwhile it's meant to get coins explicitly for use in a vendor. Making higher value coins possible in the crate just makes it easier for people to get -even more- out of it by shoving a high value coin in a slot machine, getting a refund, and getting dozens upon dozens of extra coins.

Honestly I'm not sure if the slot machine dumps out iron or silver coins, but either way, vendors only take one coin of any value for their extra content. (AFAIK).

Trilbyspaceclone commented on code/modules/cargo/packs/service.dm:
```diff
@@ -83,8 +70,103 @@
+	desc = "Contains all the essentials for an advanced spacefaring cleanup crew. This kit includes a trashbag of holding, an advanced mop, a bottle of space cleaner, a floor buffer, and a holosign projector."
+	cost = 5000
+	access = ACCESS_JANITOR
+	contains = list(/obj/item/storage/bag/trash/bluespace,
```
Hmm...

Owai-Seek replied:
Is it the access, cost, or the trashbag? I can unlist it from janitor and make it a bit more expensive, or replace the bag with a normal one.

In all honesty, most of the time when I play janitor I don't even need a bluespace bag unless someone is throwing glowsticks everywhere.

Owai-Seek committed:
Fixed Meat Crates

Owai-Seek committed:
Spellchecking and Crate balance.

Integrated #9939 spellchecking.

Rebalanced Emergency Medical Supplies, EMS Lite, and advanced janitorial crate. (Also added to Advanced Janitorial and Power Tools crate that they require access.)

Owai-Seek committed:
Removed Power Tools, Tweaked EF Crate

Removed Power Tool Crate

Made exotic footwear non-contraband, and increased price.

Owai-Seek committed:
Updated Access, Bluespace Trash (again). Fixes

Updated Engineering and Medical hardsuits to require access for their applicable departments.

Moved Virus Containment into it's proper file.

Put Bluespace Trashbag back in the advanced sanitation crate.

Spellcheck.

Owai-Seek committed:
Rebalanced Meat Crate

Owai-Seek committed:
Additional Cleanup

Reorganised Materials, added Bulk Cotton Crate
Removed Wooden Barrel / Toner Crates
Modified Fruit Delux Crate
Modified Cargo Supplies Crate
Moved several crates to their proper areas.

Owai-Seek committed:
Merge branch 'master' into cargocleanupv2boogaloo

Owai-Seek committed:
Merge branch 'master' into cargocleanupv2boogaloo

Owai-Seek committed:
Organisation
